imports/plugins/core/orders/client/components/orderSummary.js

```diff
@@ -86,28 +113,28 @@ class OrderSummary extends Component {
             <div className="order-summary-form-group">
               <strong data-i18n="order.processor">Processor</strong>
               <div className="invoice-details">
-                {order.billing[0].paymentMethod.processor}
+                {getBillingInfo(order).paymentMethod.processor}
```

Comment:
If we fall back to returning `{}` in getBillingInfo / getShippingInfo, this will throw an error here in that case. Might be worth building a few variables that won't throw errors before getting into the render function.